extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html

```diff
@@ -21,10 +21,14 @@ <h4>REST Resources</h4>
  </div>
  {/if}
  {/for}
-</header>
-
-<header>
-
-</header>
+<br>
+ <h5>Additional Endpoints</h5>
+ {#for additionalEndpoint in info:additionalEndpointInfo}
+ <div class="row">
+  <div class="col">
+   <code>{additionalEndpoint.endpoint}   {additionalEndpoint.description}</code>
```

Comment:
Could you add a link to the endpoint please? And probably use the description as the link label if there is not. If not, use the endpoint url.

Reply:
ok, I will add a link to the endpoint.

Reply:
And one more doubt I had is in `NotFoundPageDisplayableEndpointBuildItem` , we have two fields only `endpoint` and `description`. 
Umm to add a link to endpoint , like what exactly I need to put inside `href` ..

Reply:
> And one more doubt I had is in `NotFoundPageDisplayableEndpointBuildItem` , we have two fields only `endpoint` and `description`.
> Umm to add a link to endpoint , like what exactly I need to put inside `href` ..

@FroMage need help here